Source/WebCore/rendering/updating/RenderTreeUpdater.cpp

```diff
@@ -760,6 +761,13 @@ static std::optional<DidRepaintAndMarkContainingBlock> repaintAndMarkContainingB
     }
 
     for (auto it = composedTreeDescendantsIterator.begin(), end = composedTreeDescendantsIterator.end(); it != end; ++it) {
+        if (it->renderer()) {
+            auto& renderer = *it->renderer();
```

Comment:
We shouldn't be calling renderer() twice like this.
Just assign renderer to a local variable like this:
```
auto* renderer = it->renderer();
if (!renderer)
    continue;
```